src/App.js

```diff
@@ -0,0 +1,89 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { AiOutlineMenuUnfold } from 'react-icons/ai';
+
+// Pages
+import HomePage from './pages/Home/Home';
+import PlaneDetailsPage from './pages/detailsPage/planeDetailsPage';
+import AddPlane from './pages/AddPlane/AddPlane';
+import Reservations from './pages/reservations/Resvervations';
+import PlanesReservations from './pages/reservations/PlanesReservation';
+
+// Auth-Pages
+import Register from './pages/auth/Register/Register';
+import Login from './pages/auth/Login/Login';
+import AuthLayout from './pages/auth/Layout/Layout';
+
+// Components
+import AdminRoute from './components/ProtectedRoute/AdminRoute';
+import ProtectedRoute from './components/ProtectedRoute/ProtectedRoute';
+import ErrorMessage from './components/messages/ErrorMessage';
+import NoticeMessage from './components/messages/NoticeMessage';
+import Navbar from './components/Navbar/Navbar';
+import { clearAll } from './slices/appSlice/appSlice';
+
+// Stylesheet
+import './App.css';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/231818610-a73641f7-8998-41ef-88a8-1e8ec4466a26.png)

That way, your app will meet the latest standards. 💪